packages/shared/src/components/icons/Hot/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Icon, { IconProps } from '../../Icon';
+import OutlinedIcon from './outlined.svg';
+import FilledIcon from './filled.svg';
+
+const HotIcon: React.VFC<IconProps> = ({ filled = false }) => (
```

Comment:
Kindly check my concern regarding the `props` for easier maintenance. Tagging for visibility. Thank you!